http_client.go

```diff
@@ -196,7 +202,6 @@ func newSingleHostClient(timeout time.Duration) *http.Client {
 			DialContext: (&net.Dialer{
 				Timeout:   5 * time.Second,
 				KeepAlive: 30 * time.Second,
-				DualStack: true,
```

Comment:
Why is this option removed?

Reply:
It's deprecated and enabled by default according to the docs:
```
// DualStack previously enabled RFC 6555 Fast Fallback
// support, also known as "Happy Eyeballs", in which IPv4 is
// tried soon if IPv6 appears to be misconfigured and
// hanging.
//
// Deprecated: Fast Fallback is enabled by default. To
// disable, set FallbackDelay to a negative value.
DualStack bool
```
Source: https://pkg.go.dev/net#Dialer